src/main/webapp/assets/modules/contest-detail/contestDetailDirective.js

```diff
@@ -94,9 +106,9 @@ techlooper
       restrict: "E",
       replace: true,
       templateUrl: "modules/contest-detail/contestDetailReviewSubmission.html",
-      link: function (scope, element, attr, ctrl){
-        scope.goToSubmissionLink = function(link){
-          var url = (link.indexOf("https://") >= 0 || link.indexOf("http://") >= 0) ? link : "http://"+link;
+      link: function (scope, element, attr, ctrl) {
```

Comment:
'ctrl' is defined but never used.<br>'attr' is defined but never used.<br>'element' is defined but never used.